server/homebrew.api.spec.js

```diff
@@ -382,7 +381,68 @@ describe('Tests for api', ()=>{
 			await expect(fn(req, null, next)).rejects.toEqual({ 'HBErrorCode': '51', 'brewId': '1', 'brewTitle': 'test brew', 'code': 404, 'message': 'brew locked' });
 		});
 	});
+	describe('Get CSS', ()=>{
```

Comment:
Is there a particular reason some of the test cases have been moved around? It's ok if there is. It just isn't clear if it's relevant to deleting authors or if there is something special I should be checking here.